referenceframe/primitives.go

```diff
@@ -64,10 +64,10 @@ type InputEnabled interface {
 	GoToInputs(context.Context, ...[]Input) error
 }
 
-// InterpolateInputs will return a set of inputs that are the specified percent between the two given sets of
+// interpolateAllInputs will return a set of inputs that are the specified percent between the two given sets of
```

Comment:
[opt] rename `interpolateInputs`? The "all" seems redundant

Reply:
Yeah good call